tests/suites/relations/task.sh

```diff
@@ -16,6 +16,5 @@ test_relations() {
 	test_relation_data_exchange
 	test_relation_departing_unit
 	test_relation_list_app
-
```

Comment:
Drop this change I would, will make the diff easier to read

Reply:
Oh yeah forgot to drop it. I was using this as an example. probably generated the diff by mistake.